cloudant-client/src/main/java/com/cloudant/client/api/query/QueryBuilder.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2017 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.cloudant.client.api.query;
+
+public class QueryBuilder {
+
+    OperationOrExpression query;
+
+    public QueryBuilder(OperationOrExpression query) {
+        this.query = query;
+    }
+
+    public String build() {
```

Comment:
Maybe we could reduce the boiler plate even further by making this a static, since I don't think we actually need the `query` to be stored in an instance

```java
public static String build(OperationOrExpression query) {
    return String.format("{\"selector\": {%s}}", query.toString());
}
```

I know naming isn't final, but we could also possibly call the method `selector` to perhaps indicate better how it maps to the HTTP API and maybe drop the `Builder` suffix (if we don't have an explicit `build` method anyway) then:
`new QueryBuilder(gt("year", 2018)).build());`
could be
`Query.selector(gt("year", 2018));`

It's not a massive saving, but is a bit more concise.